lib/Eventz.php

```diff
@@ -96,14 +96,17 @@ public static function normalize_event( $event ) : array {
             $image = $event->images->imageMobile->url;
         }
 
+        $is_recurring = isset( $event->event->dates ) ? count( $event->event->dates ) > 1 : ( isset( $event->event->entries ) ? count( $event->event->entries ) >= 1 : null );
+
```

Comment:
Luettavuuden ja jälkeenpäin koodiin palaamisen takia purkaisin tämän ehdottomasti if-lohkoihin ja pistäisin kommentit.

```suggestion
// Not recurring by default.
$is_recurring = false;

// Do the if blocks.
...
```